e2e/pages/citizen/createCase/C100/reasonableAdjustments/ReasonableAdjustmentsNeedsDuringCourtHearingPage.ts

```diff
@@ -0,0 +1,181 @@
+import { Page } from "@playwright/test";
+import { Selectors } from "../../../../../common/selectors";
+import { CommonStaticText } from "../../../../../common/commonStaticText";
+import AxeTest from "../../../../../common/accessibilityTestHelper";
+import { ReasonableAdjustmentsNeedsDuringCourtHearingContent } from "../../../../../fixtures/citizen/createCase/C100/reasonableAdjustments/ReasonableAdjustmentsNeedsDuringCourtHearingContent";
+import { Helpers } from "../../../../../common/helpers";
+
+export interface ReasonableAdjustmentsNeedsDuringCourtHearingPageOptions {
+  page: Page;
+  accessibilityTest: boolean;
+  errorMessaging: boolean;
+  yesNoReasonableAdjustments: boolean;
+}
+
+enum ChecklistUniqueSelectors {
+  appropriateLighting = "#ra_feelComportable",
+  regularBreaks = "#ra_feelComportable-2",
+  SpaceToGetUpAndMove = "#ra_feelComportable-3",
+  Other = "#ra_feelComportable-4",
+}
+
+enum TextBoxUniqueSelectors {
+  appropriateLightingInput = "#ra_appropriateLighting_subfield",
+  otherInput = "#ra_feelComportableOther_subfield",
+}
+
+const noToAll: string = "#ra_feelComportable-6";
+
+export class ReasonableAdjustmentsNeedsDuringCourtHearingPage {
+  public static async reasonableAdjustmentsNeedsDuringCourtHearingPage({
+    page: page,
+    accessibilityTest: accessibilityTest,
+    errorMessaging: errorMessaging,
+    yesNoReasonableAdjustments: yesNoReasonableAdjustments,
+  }: ReasonableAdjustmentsNeedsDuringCourtHearingPageOptions): Promise<void> {
+    await this.checkPageLoads({
+      page: page,
+      accessibilityTest: accessibilityTest,
+    });
+    if (errorMessaging) {
+      await this.triggerErrorMessages({ page: page });
+    }
+    await this.fillInFields({
+      page: page,
+      yesNoReasonableAdjustments: yesNoReasonableAdjustments,
+    });
+  }
+
+  private static async checkPageLoads({
+    page: page,
+    accessibilityTest: accessibilityTest,
+  }: Partial<ReasonableAdjustmentsNeedsDuringCourtHearingPageOptions>): Promise<void> {
+    if (!page) {
+      throw new Error();
```

Comment:
Can you add an error message like 'page argument can't be null'? SHould help more if tests fail for this reason